README.md

```diff
@@ -320,5 +201,120 @@ This request will delete webhook permanently
 #### Refresh webhook
 If the number of failed requests wil exceed `WEBHOOK_MAXTRIES`, webhook will be set to inactive. To refresh webhook you can use this same endpoint as for webhook creation.
 
+### Running from source
+
+1. Install Go according to the installation instructions here: http://golang.org/doc/install
+
+Options to run Pulse:
+
+a) Clone the repo
+
+   ```sh
+  git clone https://github.com/bitcoin-sv/pulse
+   ``` 
+1. ```go run ./cmd/main.go```
+
+Or run app with docker
+1. ```docker compose up --build```
+
+b) Get package from ``pkg.dev.go``
+1. ```go get -u https://pkg.go.dev/github.com/bitcoin-sv/pulse```
+2. ```go build -o pulse```
+3. ```./pulse```
+
+
+### Config Variables
+
+Default config variables can be overridden by (in this order of importance):
+
+1. Flags (only the ones below)
+2. ENV variables
+3. Config file
+
+#### Flags
+
+Available flags:
+
+```bash
+  -C, --config_file string                       custom config file path
+  -h, --help                                     show help
+  -v, --version                                  show version
+  -d, --dump_config                              dump config to file, specified by config_file (-C) flag
+  -e, --export_headers                           export headers to file
+```
+
+To generate config file with defaults, use the --dump flag, or:
+
+```bash
+go run ./cmd/main.go -d
+```
+
+The default config file path is **program root**, and the default file name is **config.yaml**. This can be overridden by -C flag.
+
+```bash
+go run ./cmd/main.go -C /my/config.yaml
+```
+
+#### Environment variables
+
+To override any config variable with ENV, use the "pulse\_" prefix with mapstructure annotation path with "_" as a delimiter in all uppercase. Example:
+
+Let's take this fragment of AppConfig from `config.example.yaml`:
+
+```yaml
+websocket:
+  # Maximum number of history items
+  history_max: 300
+  # History time-to-live
+  history_ttl: 10
+```
+
+To override history_max in websocket config, use the path with "_" as a path delimiter and pulse\_ as prefix. So:
+
+```bash
+PULSE_HISTORY_MAX=300
+```
+
+### Configuration
+In the ```config.go``` is the configuration of the application. By changing variables you can adjust the work of our server
+
+```
+defaultConfigFilename          = "p2p.conf"
```

Comment:
I would remove all this section, and maybe copy-paste here the config with comments from config.example.yaml, and then we would have the config documentation of Pulse done (BUX-507), but maybe talk with @pawellewandowski98 about it, as he's assigned to this task.

Reply:
Removed